src/mesh/interpolation.cxx

```diff
@@ -146,7 +146,10 @@ const Field3D interp_to(const Field3D &var, CELL_LOC loc, REGION region) {
           // coordinates
 
           Field3D var_fa = fieldmesh->toFieldAligned(var);
-          Field3D result_fa = fieldmesh->toFieldAligned(result);
+          Field3D result_fa(fieldmesh);
+          if (loc != RGN_NOBNDRY) {
```

Comment:
shouldn't it be `region` rather then `loc`?

Reply:
yep, sorry. I must need a coffee...